src/style.css

```
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+}
+
+.nav__item__link {
+  padding: 22.5px 0;
+  margin: 0 10px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__item__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  background: #00acdc;
+  left: 0;
+  bottom: 4px;
+  width: 100%;
+  height: 4px;
+  border-radius: 20px;

```

```suggestion
  border-radius: 8px;

```
By design it should be 8px, not 20